omero/developers/cli/extending.rst

```diff
@@ -1,10 +1,12 @@
 Extensions
 ----------
 
-Plugins can be written and put in the ``lib/python/omero/plugins``
-directory. On execution, all plugins in that directory are registered
-with the |CLI|. Alternatively, the :option:`--path` argument can be used to
-point to other plugin files or directories.
+Plugins can be written and put in any directory called ``.../omero/plugins``.
+Add this directory to your ``PYTHON`` path and on execution all plugins in that
```

Comment:
Bit ambiguous - is "this" directory the "omero" directory that contains "plugins" or is it the "plugins" directory we need in the path and our code checks that its parent is named "omero"?